src/components/report-problem-issue-type/report-problem-issue-type.container.js

```diff
@@ -2,13 +2,14 @@
 import { connect } from 'react-redux';
 import { bindActionCreators } from 'redux';
 import ReportProblemIssueType from './report-problem-issue-type.component';
+import { cacheLastAnnotation } from '../../actions/trail-annotations.action-creator';
 
-const mapStateToProps = ({userLocation}) => {
-  return { userLocation };
+const mapStateToProps = ({photo, annotations}) => {
+  return { photo, annotations };
 };
 
 const mapDispatchToProps = (dispatch) => {
-  return bindActionCreators({
+  return bindActionCreators({cacheLastAnnotation
```

Comment:
The cacheLastAnnotation will take an annotation and store it on the redux state tree (but not save it to firebase).  This is the method we talked about that builds up an annotation object as you move between screens.  There's a corresponding cache method that caches the photo when it saves.

Let me know if you can think of a better name for the method.